source/_integrations/homekit_controller.markdown

```diff
@@ -182,3 +182,7 @@ In these cases, HomeKit Controller will skip polling to avoid a buildup of back
 ### I can't see any events generated for "stateless" accessories
 
 This is expected. The only way to use stateless accessories like some doorbells, buttons or remotes with Home Assistant is through device automations. Home Assistant doesn't create duplicate events for device automation triggers, so for example you won't be able to watch them with the events developer tools.
+
+### Home Assistant can't see my homebridge device (≥2021.11.x)
```

Comment:
I'd prefer not to put a version restriction here. HAP-NodeJS (and via it, homebridge) stopped supporting Bonjour-HAP in 0.8. That was released 14 Sep 2020 (https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.8.0). Even before ciao support was added the deficiencies described existed in Homebridge. Crucially, whilst older versions worked for you, the advice to switch to ciao is a valid debugging step for any Homebridge discovery related issue.

Reply:
Got it. Version removed and Homebridge capitalized.